modules/administration-guide/partials/assembly_authenticating-users-on-3rd-party-services.adoc

```diff
@@ -0,0 +1,16 @@
+// securing-che
```

Comment:
how this comment is used? In other files it seems to be similar to ID value, but not for this file

Reply:
Not sure. Some files containint it, some not. Hope Doc team will clarify.

Reply:
This is a leftover of an old file structure, it can be safely removed. 
In these comments, we usually indicate all of the parent assemblies that include these files. There used to be a Securing Che parent assembly which included both Authenticating users and Authorizing users.